compose/material3/material3/api/current.txt

```diff
@@ -124,8 +124,8 @@ package androidx.compose.material3 {
   }
 
   public static final class BottomAppBarState.Companion {
-    method public androidx.compose.runtime.saveable.Saver<androidx.compose.material3.BottomAppBarState,? extends java.lang.Object!> getSaver();
-    property public final androidx.compose.runtime.saveable.Saver<androidx.compose.material3.BottomAppBarState,? extends java.lang.Object!> Saver;
+    method public androidx.compose.runtime.saveable.Saver<androidx.compose.material3.BottomAppBarState,? extends java.lang.Object?> getSaver();
```

Comment:
I see no code changes. is it a change in tooling?

Reply:
It looks like this. Anyway, it is better to merge to avoid conflicts.

```
git diff androidx/compose-material3/1.3.0-beta04 androidx/compose-material3/1.3.0-beta03 -- compose/material3 > 1.patch
```
![image](https://github.com/user-attachments/assets/93a2a2ff-4598-4f28-88c1-7ba71e3068c8)

https://developer.android.com/jetpack/androidx/releases/compose-material3#1.3.0-beta04